tests/lava/proc/lif/test_models.py

```diff
@@ -386,6 +386,31 @@ def test_bitacc_pm_impulse_dv(self):
         self.assertListEqual(expected_v_timeseries, lif_v)
         self.assertListEqual(expected_float_v, lif_v_float.tolist())
 
+    def test_bitacc_pm_eff_bias(self):
```

Comment:
Can we call this something like `test_bitacc_pm_scaling_of_bias` to make it more explicit?

Reply:
Thanks for the comment, addressed in 8fa7f87.